README.md

```diff
@@ -205,16 +259,29 @@ The following is merely on overview over the most important parameters.
 
 ## Example Call
 
-* Change the basic stuff like inputbase and outputbase directories
-set RUN_FEATURE_COUNTS_DEXSEQ, RUN_RNASEQC, RUN_KALLISTO, RUN_ARRIBA, runFingerprinting to FALSE.
-* Check if your sample type is in possibleTumorSampleNamePrefixes
-G Set your star index (GENOME_STAR_INDEX) and gene models (GENE_MODELS) parameters to what your created for (1) and (2).
-* Set GENOME_GATK_INDEX to point to the new FASTA file (1), for which you have created a dict file. The dict file should be in the same directory as the FASTA.
+* Change the basic stuff like `inputbase` and `outputbase` directories
+set `RUN_FEATURE_COUNTS_DEXSEQ`, `RUN_RNASEQC`, `RUN_KALLISTO`, `RUN_ARRIBA`, `runFingerprinting` to `FALSE`.
+* Check if your sample type is in `possibleTumorSampleNamePrefixes`
+* Set your star index (`GENOME_STAR_INDEX`) and gene models (`GENE_MODELS`) parameters to what your created for (1) and (2).
+* Set `GENOME_GATK_INDEX` to point to the new FASTA file (1), for which you have created a dict file. The dict file should be in the same directory as the FASTA.
 
 ## Change Log
 
-* 3.1.0
-  - patch: Updated default from subread 1.5.1 to 1.6.5. The previous version produces occasional segmentation faults (related to extreme optimization option `-O6`) but otherwise produces the same results. Both versions produced exactly the same `featureCounts` in multiple tests.
+* 4.0.0
+  - major: Arriba 2.2.1
+  - major: `resources/configurationFiles/analysisRNAseq.xml` is now just a default configuration with many configuration options left blank. For your `<analysis>` tags in your project XMLs the analysis names can be changed to use the following defaults:
+     * GRCh37, mm10: [RNAseqAnalysis](resources/configurationFiles/analysisRNAseq.xml)
+     * GRCh38-specific: [RNAseqAnalysisGRCh38](resources/configurationFiles/analysisRNAseqGRCh38.xml)
+  - major: Update default software versions for all assemblies
+    * STAR 2.7.10a (from 2.5.3a for hg37/mm10 and 2.7.6a for hg38)
+    * Kallisto 0.46.0 (from 0.42.0)
+    * Samtools 1.9 (from 1.6)
+    * HTSlib 1.9 (from 1.6)
+    * subread 1.6.5 (from 1.5.1): The previous version produces occasional segmentation faults (related to extreme optimization option `-O6`) but otherwise produces the same results. Both versions produced exactly the same `featureCounts` in multiple tests.
+  - major: Gene model gencode version 31 was used to create the STAR and Kallisto indexes
```

Comment:
The gencode version was updated to v39 in the xenograft reference genome. Will this also be done here as well? Then the STAR indices file paths should be updated in both GRCh37 and GRCh38 XML configurations.

Reply:
Furthermore I added
* https://odcf-gitlab.dkfz.de/ODCF/refmake/-/merge_requests/37
which will create
* `/omics/odcf/reference_data/by-species/Homo_sapiens/GRCh37_p13/GRCh37_p13_decoy_phiX`

Reply:
And (GRCh38 without ALT/HLA; the previous PR had to be split, because there is apparently a bug (race condition) with concurrently calculating the assemblies using the same input files):
* https://odcf-gitlab.dkfz.de/ODCF/refmake/-/merge_requests/38